aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -725,6 +725,34 @@ Tensor& mvlgamma_(Tensor& self, int64_t p) {
   return self.copy_(args.lgamma_().sum(-1).add_(p2_sub_p * std::log(c10::pi<double>) * QUARTER));
 }
 
+Tensor& mvlgamma_out(const Tensor& self, int64_t p, Tensor& result) {
+  TORCH_CHECK(
```

Comment:
These device checks are now autogenerated (unless opted out of in native_functions.yaml), so I don't think we need this check anymore

Reply:
Thanks!